notesapi/v1/tests/test_models.py

```diff
@@ -20,6 +20,7 @@ def setUp(self):
                     "endOffset": 10,
                 }
             ],
+            "tags": [u"apple", u"pear"],
```

Comment:
Must: why add a non-empty "tags" list here if the only test in this file which exercises tags removes it?